docs/tools/vdb_table/data/vespa.json

```diff
@@ -73,23 +73,23 @@
     "comment": ""
   },
   "embeddings_image": {
-    "support": "none",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/vespa-engine/sample-apps/tree/master/text-image-search",
```

Comment:
Thanks for linking the explanation - do I read it correctly that you require the user to pull down CLIP, modify it, bundle it as ONNX and then you use the ONNX inference hosting within vespa cloud and maintain the clip_export.py code on their side as new CLIP versions come out for example?

Going with the full/partial/none definitions below, this would count as "partial" support.

Reply:
Thanks for clarification of the definitions. 
I can agree on partial, until we provide more documentation and examples on how this can be done more smoothly.